discovery-infra/consts.py

```diff
@@ -14,11 +14,11 @@
 NUMBER_OF_MASTERS = 3
 TEST_INFRA = "test-infra"
 CLUSTER = "%s-cluster" % TEST_INFRA
-CLUSTER_PREFIX = "%s-" % CLUSTER
+CLUSTER_PREFIX = "%s" % CLUSTER
 TEST_NETWORK = "%s-net" % TEST_INFRA
```

Comment:
or delete it or use it. I like TEST_NETWORK as name more cause we have many networks here :)